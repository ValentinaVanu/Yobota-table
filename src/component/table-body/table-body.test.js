import {YobotaTableBody} from '.'
import renderer from 'react-test-renderer'

test('renders yobota table body component', () => {
  const component = renderer.create(<YobotaTableBody />).toJSON()
  expect(component).toMatchSnapshot()
})
