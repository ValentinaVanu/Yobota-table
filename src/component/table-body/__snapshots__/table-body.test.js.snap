// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders yobota table body component 1`] = `
<tbody
  className="MuiTableBody-root"
  role={null}
/>
`;
